next-i18next.config.js

```diff
@@ -1,6 +1,12 @@
+// @ts-check
```

Comment:
Not sure about a TypeScript check on a JavaScript file...

Reply:
Information about @ts-check
https://www.typescriptlang.org/docs/handbook/intro-to-js-ts.html#ts-check